scripts/buildhost/s3streamup.sh

```diff
@@ -13,31 +13,30 @@ DEST=$1
 NO_SIGN=$2
 
 SEQ=1
+aws s3 rm ${DEST}-${SEQ} ${NO_SIGN}
+aws s3 rm ${DEST}-$((SEQ+1)) ${NO_SIGN}
 SEND_NEXT=$((SECONDS+10))
 BUFFER=
 while read line; do
         BUFFER=${BUFFER}${line}$'\n'
         if [ ${#BUFFER} -gt 10240 ]; then
-                NEXT_SEQ=$((SEQ+1))
```

Comment:
not sure what happens here

Reply:
instead of "delete log X+1, write to log X", I changed the logic to "write to log X, delete log X+2".